### ISSUE TITLE
As a supervisor, I only want to see Open and Completed Conversations with my current employees

### ISSUE THREAD

**Issue by Travis-A-Clark** [bcgov/performance#889]
A supervisor should only be able to access open and completed conversations with their current employees. The conversations should always remain accessible to the employees themselves.

For example:
- Supervisor A completes a conversation with Employee B in October.
- Supervisor A moves to a new position in November, meaning that Employee B is no longer on their team.
- Supervisor A should no longer see any of their open or completed conversations with Employee B under the "Conversations with my Team" area. This should be populated by only the same employees that Supervisor A sees on their "My Team" page.
- Employee B should ALWAYS see the conversation under the "Conversations with my Supervisor" heading; even after the supervisor has changed.

Basically, the conversation belongs to the employee and they should always see it. The supervisor should only have access to the conversation so long as the employee remains one of their direct or shared reports.

**Comment by jizhaogit** [comment]
@telusdcinco code is ready for you to review. thanks.

**Comment by telusdcinco** [comment]
@jizhaogit Please proceed to migrate your fix.

**Comment by jizhaogit** [comment]
@telusdcinco thank you!

**Comment by SSubashree26** [comment]
@jizhaogit I noticed the following observations while creating the test data for this story

Observation 1:
The "Invalid Employee ID" error message is displayed incorrectly at two places
1 - When the Supervisor signs off with an incorrect employee ID, the error message is displayed in the Employee Sign-Off column too
![Screenshot 2023-01-24 at 5.18.18 PM.png](https://images.zenhubusercontent.com/63892df90d58261296a2b5c0/32bd73f5-4c3a-4c28-9a4a-bb3a2184bf64)
2. When the employee is signing with an incorrect employee ID, after clicking the checkbox "Team member disagrees with the information contained in this performance review", the "Invalid Employee ID" error message is displayed below the disagreeing content 
![Screenshot 2023-01-24 at 5.22.13 PM.png](https://images.zenhubusercontent.com/63892df90d58261296a2b5c0/cf120014-7fa4-4f74-af3a-80592849809a)


Observation 2:
When the sysadmin1 user tries to access the Open Conversations, they are displayed with the "500 - Server error" message
https://images.zenhubusercontent.com/329107952/eee51d30-d337-4249-8d6f-5b20fffe6078/500___server_error.mov

Can you please have a look at it?

**Comment by jizhaogit** [comment]
#1 fixed a display error for inputting invalid employee id
#2 fixed query when login as sysadmin

**Comment by jizhaogit** [comment]
@telusdcinco Please help review. thanks!

**Comment by telusdcinco** [comment]
@jizhaogit Please proceed with the changes to Test.

**Comment by SSubashree26** [comment]
Completed conversations are still visible for the supervisor and therefore Ji is working on it to fix it. Moving it to "In progress" column

**Comment by SSubashree26** [comment]
Validated this story with Ji. The supervisor is not able to view both the open and the completed conversations of the previous employees. Moving this to PO review
